e2e-pw/ci/Dockerfile

```
@@ -22,15 +24,15 @@ RUN apt-get update \
+    /uv /uvx /usr/local/bin/
 
-# Bake fiftyone's dependency tree exactly as setup.py resolves it, then drop
-# fiftyone itself — the shard job reinstalls it from the PR's source, which
-# is fast once the dependencies are already present.
+# Bake the locked dependency tree without the project itself. The shard job
+# syncs the PR checkout into the same stable environment.
 COPY . /tmp/fiftyone-src
-RUN uv pip install --system '/tmp/fiftyone-src[multimodal]' \
-        -r /tmp/fiftyone-src/requirements/e2e.txt \
-    && uv pip uninstall --system fiftyone \
+RUN cd /tmp/fiftyone-src \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)